raiden/tests/utils/eth_node.py

```diff
@@ -39,7 +29,7 @@
 
 
 class EthNodeDescription(NamedTuple):
-    private_key: PrivateKey
```

Comment:
What's wrong with the PrivateKey type?

Reply:
It was wrong, it is `bytes` but the lack of typing in the surround code didn't catch it.